tts/global.def.lua

```diff
@@ -1,3 +1,10 @@
+
+---@overload fun(label: string, callback:fun(label: string, object: tts__Object, pointerPosition: nil | tts__Vector, isKeyUp: boolean): void`)
```

Comment:
On mobile, so it may just be displaying weirdly. However, looks like there's a stray backtick near the end of this line. If you could throw a sieve before `fun` that'd be great too.

If you happen to know the return type for this method, it's be great if you could add it to the function and the overload. If not, that's fine, can always be added later. A partial (but still accurate) type is far superior to no type at all 😉

Reply:
Backtick fixed.  Good eye.  As for the return type? Nope.  The patch notes that talk about it aren't that detailed